src/Sarif/Baseline/ResultMatching/DataStructures/MatchingResult.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Sarif.Baseline.ResultMatching
+{
+    public class MatchingResult
```

Comment:
>public class MatchingResult [](start = 4, length = 27)

This almost certainly does not need to be public.

Reply:
Comment incorrect, this is part of multiple public interfaces.

---
In reply to: [211699406](https://github.com/Microsoft/sarif-sdk/pull/1034#discussion_r211699406) [](ancestors = 211699406)